tests/models/marian/test_tokenization_marian.py

```diff
@@ -149,3 +149,10 @@ def test_tokenizer_integration_seperate_vocabs(self):
 
         decoded = tokenizer.decode(target_ids, skip_special_tokens=True)
         self.assertEqual(decoded, target_text)
+
+    def test_tokenizer_decode(self):
+        tokenizer = MarianTokenizer.from_pretrained("Helsinki-NLP/opus-mt-en-es")
+        source_text = "_This is 1 text string that starts with an _ and ends with one too _"
+        ids = tokenizer(source_text)["input_ids"]
```

Comment:
Instead of manually adding the spiece underline we can just use the example from the issue: `tokenizer.decode(tokenizer("hello world")['input_ids'], skip_special_tokens=True)`

Reply:
+1

Also, I can't quite tell on mobile, but are those just regular underscores? The sentencepiece underscore is a slightly different character (but looks similar).

Reply:
My fault. My VSCode theme really made them both look similar 🥲
Fixed it now!